tracking/src/Tracker.js

```diff
@@ -0,0 +1,126 @@
+const fetch = window.fetch;  
+const URI = require('jsuri'); 
+const pjson = require('../../package.json')
+const BASE_URL = "https://analytics.data.visenze.com/v3";
+const BASE_URL_CN = "https://analytics.visenze.com.cn/v3";
+const SDK = "javascript sdk"; 
+const SDK_VERSION = pjson.version; 
+const TIMEOUT = 15000;
+
+
+/**
+* Adds a list of query parameters
+* @param  {params}  params object
+* @return {URI}     returns self for fluent chaining
+*/
+URI.prototype.addQueryParams = function (params) {
```

Comment:
can prefix the addQueryParams ? just in case we have a conflict with another library.

Reply:
changed